modules/custom-css/custom-css/js/core-customizer-css.js

```diff
@@ -0,0 +1,199 @@
+(function( wp, $, api ){
+	api.controlConstructor.jetpackCss = api.Control.extend({
+		modes: {
+			'default': 'text/css',
+			'less': 'text/x-less',
+			'sass': 'text/x-scss'
+		},
+		_updating: false,
+		/**
+		 * Fires when our control is ready for action. Gets everything set up.
+		 * @return {null}
+		 */
+		ready: function() {
+			this.opts = window._jp_css_settings;
+			// add our textarea
+			this.$input = $( '<textarea />', {
+				name: this.setting.id,
+				'class': 'for-codemirror hidden'
+			} ).val( this.setting() );
+			this.container.append( this.$input );
+
+			// keep the textarea and the setting synced up
+			api( this.setting.id, _.bind( function( setting ){
+				var element = new api.Element( this.$input );
+				this.elements = [ element ];
+				element.sync( setting );
+				element.set( setting() );
+			}, this ) );
+
+			// should we use CodeMirror?
+			if ( this.opts.useRichEditor ) {
+				this.initCodeMirror();
+			} else {
+				this.$input.removeClass( 'hidden' );
+			}
+
+			api.bind( 'ready', _.bind( this.addLabels, this ) );
```

Comment:
I have a vague recollection of needing this action delayed, but I wrote this code a long time ago. Swapping to `this.addLabels()` would be fine if it works.